src/main/java/org/spongepowered/api/world/World.java

```diff
@@ -262,6 +262,14 @@ default LocatableBlock getLocatableBlock(int x, int y, int z) {
     WorldBorder getWorldBorder();
 
     /**
+     * Sets the world border for this world.
+     * 
+     * @param border The world border to be used.
+     * @param cause The cause of the world border's change.
+     */
+    void setWorldBorder(WorldBorder border, Cause cause);
```

Comment:
If you want, you can add a method to `WorldBorder` to apply the values from another `WorldBorder` instance (e.g. `WorldBorder.apply(WorldBorder)`. I don't think it makes sense to have such a method on `World`